### PR TITLE
M3 4608 test updates for ci runs

### PR DESCRIPTION
**Pull request by ndukatz**
**How to verify that that the test works:**
- Make sure mock service worker is off.
- yarn up
- yarn cy:e2e **(runs headless)**
- Result should be "✔  All specs passed!"

Make sure your .env contains:
REACT_APP_LOGIN_ROOT='https://login.linode.com'
REACT_APP_API_ROOT='https://api.linode.com/v4'
REACT_APP_APP_ROOT='http://localhost:3000'
REACT_APP_LAUNCH_DARKLY_ID=(CMR flag)
REACT_APP_CLIENT_ID=(https://cloud.linode.com/profile/clients, copy ID)
MANAGER_OAUTH=(create in https://cloud.linode.com/profile/tokens > "Add a Person Access Token", Save the token that appears in the modal, you will only see it once)

### CODE REVIEW

File: packages/manager/cypress/integration/firewall/migrate-linode-with-firewall.spec.ts [hunk 1]
```diff
@@ -162,47 +166,53 @@ describe('Migrate Linode With Firewall', () => {
...
           .should('eq', 400);
-      }
-      cy.findByText(
-        'Target region for this Linode does not support Cloud Firewalls at this time. Please choose a different region or remove your firewall before migrating.'
-      ).should('be.visible');
+        if (!cy.findByText('Linode busy.').should('not.be.visible')) {
+          fbtVisibleClick('Enter Migration Queue');
+          cy.wait('@migrateLinode')
+            .its('status')
+            .should('eq', 400);
```
Comment (Jskobos) [inline packages/manager/cypress/integration/firewall/migrate-linode-with-firewall.spec.ts:208]: This is failing consistently; the response status code is 200.

**Review by tiffwong — COMMENTED** [review]
I had 10/20 fail

**Review by Jskobos — APPROVED** [review]
All passing now, thanks

File: .github/workflows/e2e.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+name: Run Cypress E2E Tests
```
Comment (johnwcallahan) [inline .github/workflows/e2e.yml:1]: I'd put this workflow file in a different PR so we can talk about it in isolation. Specifically we'll want to update the `branch` field and decide on a schedule, etc.

**Review by johnwcallahan — COMMENTED** [review]
Might be what you mentioned in standup, but I'm seeing 1-2 errors each time I run the suite.

**Review by johnwcallahan — APPROVED** [review]
Nice work; all passing for me.

```
    ✔  All specs passed!                        11:59       99       73        -       26        -

✨  Done in 819.62s.
```
